packages/svelte/src/reactivity/start-stop-notifier.js

```diff
@@ -0,0 +1,40 @@
+import { tick, untrack } from '../internal/client/runtime.js';
+import { effect_tracking, render_effect } from '../internal/client/reactivity/effects.js';
+
+/**
+ * Returns a function that, when invoked in a reactive context, calls the `start` function once,
+ * and calls the `stop` function returned from `start` when all reactive contexts it's called in
+ * are destroyed. This is useful for creating a notifier that starts and stops when the
+ * "subscriber" count goes from 0 to 1 and back to 0.
+ * @param {() => () => void} start
+ */
+export function createStartStopNotifier(start) {
```

Comment:
wondering if this is the right name for this function. `StartStopNotifier` is slightly esoteric. what if it was `this.#subscriber = createSubscriber(fn)` instead? would that make sense?

Reply:
I think so - I was hesitant to adopt the subscriber nomenclature because of it sounding too much like stores, but it explains the concept much better